mk/linux/build-appimage.sh

```diff
@@ -1,15 +1,28 @@
 #!/bin/sh
 
-SCRIPTLOC=$(dirname "$0")
+set -e
 
-if [ -d $SCRIPTLOC/BuildAppImage/AppDir ]; then
-    rm -r $SCRIPTLOC/BuildAppImage/AppDir
+# SCRIPTLOC=$(dirname "$0")
+SCRIPTLOC="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+mkdir -p $SCRIPTLOC/BuildAppImage/
+cd $SCRIPTLOC/BuildAppImage/
+
+if [ ! -f linuxdeploy-x86_64.AppImage ]; then 
+    wget https://github.com/linuxdeploy/linuxdeploy/releases/download/1-alpha-20220822-1/linuxdeploy-x86_64.AppImage
```

Comment:
You want to add linuxdeploy the to build environment

Example:

https://github.com/0ad-matters/0ad-appimage/blob/0284749f021d90300c58fec658516969b5557843/Dockerfile#L45